src/amo/css/AddonReviewList.scss

```diff
@@ -0,0 +1,91 @@
+@import "~amo/css/inc/vars";
+
+.AddonReviewList {
+  padding: 10px;
+}
+
+.AddonReviewList-header {
+  margin: 15px 0;
+  display: flex;
+  flex-direction: row;
+
+  .AddonReviewList-header-text {
+    > h2,
+    > h4 {
+      font-size: 1.1em;
+    }
+
+    > h2 {
+      padding-top: 10px;
+      margin: 0 0 5px;
+    }
+
+    > h4 {
+      font-weight: normal;
+      margin: 0;
+    }
+
+    a:link,
+    a:visited,
+    a:hover,
+    a:active {
+      text-decoration: none;
+      color: $primary-font-color;
+      font-weight: normal;
+    }
+  }
+
+  .AddonReviewList-header-icon {
+    margin: 0 20px;
+  }
+}
+
+.AddonReviewList-ul {
+  list-style: none;
+  background-color: white;
+  margin: 10px 0;
+  padding: 0;
+  border-radius: 10px;
+}
+
+$star-size: 13px;
+$star-gutter: 4px;
+
+.AddonReviewList-li {
+  margin: 0;
+  padding: 10px;
+  border-bottom: 2px solid $accent-color;
+
+  &:last-of-type {
+    border-bottom: none;
+  }
+
+  > h3 {
+    padding: 0;
+    margin: 0;
+  }
+
+  > p {
+    margin: 0 0 5px;
+  }
+
+  .AddonReviewList-by-line {
+    display: flex;
+    justify-content: flex-start;
+    font-size: 0.9em;
+    color: $sub-text-color;
+  }
+
+  .Rating {
+    margin: 0 5px 0 0;
+    min-width: ($star-size + $star-gutter) * 5;
```

Comment:
I thought so at first but the stars got truncated. On second look, this was some kind of auto-collapsing or borders that `float` was doing. I switched it to flex and your math is working now. I am also moving all of these into Rating.